services/pdf-ms/helpers/case.ts

```diff
@@ -0,0 +1,55 @@
+import to from 'await-to-js';
+
+import config from '../../../config';
+import * as dynamoDb from '../../../libs/dynamoDb';
+import { Case, AnswerObject } from './types';
+
+/** Get all cases for a given person and formId */
+export const getRelevantCases = async (personalNumber: string, formId: string) => {
+  const params = {
+    TableName: config.cases.tableName,
+    IndexName: 'PK-formId-gsi',
+    KeyConditionExpression: 'PK = :pk and formId = :formId',
+    ExpressionAttributeValues: { ':pk': `USER#${personalNumber}`, ':formId': formId },
+  };
+  const [error, casesResponse] = await to<{
+    Count: number;
+    Items: {
+      answers: AnswerObject[];
+      updatedAt: number;
+      PK: string;
+      SK: string;
+      provider: string;
+      formId: string;
+      status: 'ongoing' | 'submitted';
+      details: Record<string, any>;
+    }[];
+    ScannedCount: number;
+  }>(dynamoDb.call('query', params));
+  if (error) {
+    console.error(error);
+    return undefined;
+  }
+  return casesResponse.Items;
+};
+
+/** compare two cases, returning lists of new and changed values */
+export const compareCases = (
+  currentCase: Case & {
+    answersArray: AnswerObject[];
+  },
+  oldCase: { answers: AnswerObject[] }
+) => {
+  const changedValues: string[] = [];
+  const newValues: string[] = [];
+
+  currentCase.answersArray.forEach(answer => {
```

Comment:
why forEach?

Reply:
Why not? In general I do think I find forEach cleaner than for loops, but I don't think it really matters to much which one is used.

Reply:
forEach is perfectly fine and in my opinon easier to read as-well (y)